tests/sync/courseDb.test.js

```diff
@@ -187,27 +187,6 @@ describe('course database', () => {
         assert.isOk(result.data);
       });
     });
-
-    it('warns about a default assessment set being present', async () => {
```

Comment:
This test suite wasn't being executed before; I had to fix some tests that were broken without us knowing.